README.md

```diff
@@ -3,18 +3,94 @@
 A collection of Maven plugins and utilities that can be used to build CICS bundles, ready to be installed into CICS TS.
 
 This project contains:
- - `cics-bundle-maven-plugin`, a Maven plugin that builds CICS bundles, including Java-based dependencies.
- - `cics-bundle-reactor-archetype`, a Maven archetype that shows how a simple reactor build can pull together a CICS bundle and a Java project.
+ - `cics-bundle-maven-plugin`, a Maven plugin that builds CICS bundles, in which you can include any supported bundlepart.
+ - `cics-bundle-reactor-archetype`, a Maven archetype that provides a simple reactor module that contains a CICS bundle and a Dynamic Web Project (WAR).
```

Comment:
Better, but I'd leave it as `reactor build` as a reactor is a collection of Maven modules.